packages/editor/src/client/routes/articleList.tsx

```diff
@@ -16,7 +16,7 @@ import {
 
 import {DescriptionList, DescriptionListItem} from '../atoms/descriptionList'
 
-import './theme.less'
+import '../global.less'
```

Comment:
I'm not sure if this import works. The file seems to be one directory up. But it would anyway be better if you import the `global.less` just once in the `app.tsx`